pkg/manager/manager.go

```diff
@@ -31,7 +31,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	ctrlmgr "sigs.k8s.io/controller-runtime/pkg/manager"
 
-	infrav1 "sigs.k8s.io/cluster-api-provider-vsphere/api/v1alpha3"
```

Comment:
v1alpha3 is named `infrav1` everywhere else. I’d suggest leaving it and use non import aliasing for v1alpha2 as you have done in the `main.go`

Reply:
Amended.